<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wordsMode</class>
 <widget class="QDialog" name="wordsMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>870</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>870</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>870</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>270</width>
     <height>830</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{ background-color: rgb(102, 0, 204, 100); color: rgb(255, 51, 0) }
QListWidget::item::selected{ background-color: rgb(0, 102, 204, 150); color: rgb(255, 51, 0) }
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTranslate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(255, 255, 255, 180); color: rgb(19, 121, 255)}
QPushButton::hover{background-color: rgba(255, 255, 255, 230)}</string>
   </property>
   <property name="text">
    <string>Show
Translation</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrompt3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>510</y>
     <width>310</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>&lt;--
For those functions need to select
a word from the list left
&lt;--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWordsCount">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>20</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 0, 0, 100);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChange">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(255, 191, 0, 100); color: rgb(255, 255, 0)}
QPushButton::hover{background-color: rgba(255, 191, 0, 150)}</string>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>225</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 0, 0, 100);</string>
   </property>
   <property name="text">
    <string>Amount of Words:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>790</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(255, 0, 0, 100); color: rgb(255, 102, 102)}
QPushButton::hover{background-color: rgba(255, 0, 0, 150)}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTranslator">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>20</y>
     <width>205</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(255, 255, 255, 180); color: rgb(19, 121, 255)}
QPushButton::hover{background-color: rgba(255, 255, 255, 230)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetImg">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>810</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(255, 0, 0, 100); color: rgb(255, 255, 0)}
QPushButton::hover{background-color: rgba(255, 0, 0, 150)}</string>
   </property>
   <property name="text">
    <string>Set Image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAdd">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>430</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 180);
color: rgb(255, 51, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(13, 213, 29, 100); color: rgb(153, 255, 51)}
QPushButton::hover{background-color: rgba(13, 213, 29, 150)}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrompt1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>630</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>To add a word you should write it in English first, then put the slash
with two spaces (like &quot; / &quot;), then in Russian. An exemple: &quot;hello / привет&quot;.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrompt2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>165</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>You can search
not only in English</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFind">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(19, 121, 255, 100); color: rgb(0, 255, 255)}
QPushButton::hover{background-color: rgba(19, 121, 255, 150)}</string>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRunNote">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>760</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(255, 0, 0, 100); color: rgb(255, 255, 0)}
QPushButton::hover{background-color: rgba(255, 0, 0, 150)}</string>
   </property>
   <property name="text">
    <string>Run Note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCrib">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>710</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(255, 0, 0, 100); color: rgb(255, 255, 0)}
QPushButton::hover{background-color: rgba(255, 0, 0, 150)}</string>
   </property>
   <property name="text">
    <string>Crib</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
